contract/assembly/index.ts

```diff
@@ -119,86 +88,36 @@ export function getMaxAmountPerTip(): u128 {
   return storage.get<u128>(MAX_AMOUNT_PER_TIP_KEY, u128.Zero)!;
 }
 
-// Requests
-
-export function getPendingRequests(): u32[] {
-  _active();
-  return pendingRequests.values();
-}
-
-export function getVerificationRequest(id: u32): VerificationRequest | null {
-  _active();
-  if (!verificationRequests.containsIndex(id)) return null;
-  return verificationRequests[id];
-}
-
-export function getRequestStatus(id: u32): u8 {
-  _active();
-  if (!verificationRequests.containsIndex(id)) {
-    return u8(0); // not found
-  } else if (pendingRequests.has(id)) {
-    return u8(1); // pending
-  } else if (approvedRequests.has(id)) {
-    return u8(2); // approved
-  } else {
-    return u8(3); // rejected
-  }
-}
-
 // Tipping
 
 export function getTotalTipsByItemId(itemId: string): u128 {
   _active();
   return totalTipsByItem.get(itemId, u128.Zero)!;
 }
 
-export function getTotalTipsByExternalAccount(externalAccount: ExternalAccount): u128 {
+export function getTotalTipsByAccount(accountGlobalId: AccountGlobalId): u128 {
   _active();
-  return totalTipsByExternal.get(externalAccount, u128.Zero)!;
+  return totalTipsByAccountGlobalId.get(accountGlobalId, u128.Zero)!;
 }
 
-export function getAvailableTipsByExternalAccount(externalAccount: ExternalAccount): u128 {
+export function getAvailableTipsByAccount(accountGlobalId: AccountGlobalId): u128 {
   _active();
-  return availableTipsByExternal.get(externalAccount, u128.Zero)!;
+  return availableTipsByAccountGlobalId.get(accountGlobalId, u128.Zero)!;
 }
 
 export function calculateFee(donationAmount: u128): u128 {
   _active();
   return u128.muldiv(donationAmount, u128.from(3), u128.from(100)); // 3%
 }
 
-// WRITE
-
-// Identity
-
-export function approveRequest(requestId: u32): void {
+export function getWalletForAutoclaim(accountGId: AccountGlobalId): NearAccount | null {
```

Comment:
Optional...
`getWalletForAutoclaim` => `getAccountForAutoclaim`

Reply:
Not all the accounts are wallets. Here we get only wallets. Because account for receiving tips cannot be a social network account. It's always NEAR wallet.